test/unit/definitions/divorce/AuthorisationCaseState.test.js

```diff
@@ -1,7 +1,7 @@
 const expect = require('chai').expect;
 const { uniqWith } = require('lodash');
 
-const caseTypeTab = Object.assign(require('definitions/divorce/json/AuthorisationCaseState'), {});
+const caseTypeTab = Object.assign(require('definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState'), {});
```

Comment:
Please follow the approach from this file:
test/unit/definitions/divorce/AuthorisationCaseEvent.test.js

to cover all possible scenarios.